starboard/android/shared/media_codec_bridge.cc

```diff
@@ -193,11 +194,11 @@ scoped_ptr<MediaCodecBridge> MediaCodecBridge::CreateAudioMediaCodecBridge(
       new MediaCodecBridge(handler));
   jobject j_media_codec_bridge = env->CallStaticObjectMethodOrAbort(
       "dev/cobalt/media/MediaCodecBridge", "createAudioMediaCodecBridge",
-      "(JLjava/lang/String;Ljava/lang/String;IILandroid/media/MediaCrypto;"
+      "(JLjava/lang/String;Ljava/lang/String;IILandroid/media/MediaCrypto;Z;"
```

Comment:
Please fix this with "(JLjava/lang/String;Ljava/lang/String;IILandroid/media/MediaCrypto;Z", because this cause calling `createAudioMediaCodecBridge` failed (no such static method), resulting in failing on_device tests.

Reply:
Thank you very much for finding the cause.

Reply:
Revised the patch